Scripts/script-Append.yml

```diff
@@ -0,0 +1,33 @@
+commonfields:
+  id: Append
+  version: -1
+name: Append
+script: |
+  result = demisto.args()["prefix"] + demisto.args()["suffix"]
+  res = {
+      'ContentsFormat': formats['text'],
+      'Type': entryTypes['note'],
+      'Contents': result,
+      'ReadableContentsFormat': formats['text'],
+      'HumanReadable': result,
+      'EntryContext': {demisto.args()['key']: result}
+  }
+  demisto.results(res)
+type: python
```

Comment:
If it is that important to add garbage to the war room, please provide better human readable  message to the war room, no one will understand that.